pkg/servicescm/servicescm_test.go

```diff
@@ -238,6 +238,60 @@ func TestValidateDependencies(t *testing.T) {
 			},
 			expectedErr: true,
 		},
+		{
+			name: "Cyclical dependency structure",
+			input: []Service{
+				{
+					Name:         "new-bootstrap-service",
+					Command:      "C:\\new-service",
+					Dependencies: []string{"test-controller-service"},
+					Bootstrap:    false,
+					Priority:     0,
+				},
+				{
+					Name:         "test-controller-service",
+					Command:      "C:\\test-controller-service",
+					Dependencies: []string{"test-controller-service-2"},
+					Bootstrap:    false,
+					Priority:     1,
+				},
+				{
+					Name:         "test-controller-service-2",
+					Command:      "C:\\test-controller-service-2",
+					Dependencies: []string{"new-bootstrap-service"},
+					Bootstrap:    false,
+					Priority:     2,
+				},
+			},
+			expectedErr: true,
+		},
+		{
+			name: "Disconnect cyclical dependency structure",
```

Comment:
Why disconnected? IMO, this test is equivalent to the first one (`Cyclical dependency structure`).